### PR TITLE
click 0.4.0

### PR DESCRIPTION
**Pull request by kanolato**
- [ x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/click.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Click < Formula
   desc "The command-line interactive controller for Kubernetes"
   homepage "https://github.com/databricks/click"
-  url "https://github.com/databricks/click/archive/v0.3.2.tar.gz"
+  url "https://github.com/databricks/click/archive/v0.4.0.tar.gz"
   sha256 "eed648409bf78a05658a9d097e5099ca17bf19df70122e2067859ae94c5575d5"
```
Comment (mistydemeo) [inline Formula/click.rb]: Thanks for the contribution! You'll need to update the hash in addition to the URL.

**Review by mistydemeo — CHANGES_REQUESTED** [review]

File: Formula/click.rb (hunk 1 above)
Comment (kanolato) [reply Formula/click.rb]: done! @mistydemeo

**Review by mistydemeo — APPROVED** [review]

**Comment by mistydemeo** [comment]
Would you be able to update your changes on the latest master? It looks like your branch was missing baef94a51640d5a38740ecc7e7314de29fa6924c, which led to a merge conflict.

**Comment by mistydemeo** [comment]
Instead of a merge, can you please rebase and squash it into a single commit? The CI bot works by applying your patches to the master branch, and right now the early commits in this branch don't apply.

File: Formula/click.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 class Click < Formula
   desc "The command-line interactive controller for Kubernetes"
   homepage "https://github.com/databricks/click"
-  url "https://github.com/databricks/click/archive/v0.3.2.tar.gz"
-  sha256 "eed648409bf78a05658a9d097e5099ca17bf19df70122e2067859ae94c5575d5"
+  url "https://github.com/databricks/click/archive/v0.4.0.tar.gz"
+  sha256 "69df76f57af8b0c244ef4c975a242d3fe84f8e4f4fd44e62b2ccdf24ea82b23d"
   head "https://github.com/databricks/click.git"
-
+
```
Comment (mistydemeo) [inline Formula/click.rb]: In addition to rebasing/squashing down to one commit, can you also remove this stray whitespace?

**Review by mistydemeo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**